src/Github.elm

```diff
@@ -503,3 +505,79 @@ jsonResolver decoder =
 
                 Http.BadStatus_ metadata _ ->
                     Err (Http.BadStatus metadata.statusCode)
+
+
+{-| See <https://docs.github.com/en/rest/reference/git#get-a-blob>
+
+This function returns the blob content as a base64-encoded string.
+
+NOTE: Not all output fields are supported yet. Pull requests adding more complete support are welcome.
+
+-}
+getBlobAsBase64 :
+    { repo : String
+    , file_sha : String
+    }
```

Comment:
@jxxcarlson why wasn't an `authToken` needed for getBlog?  I couldn't really find an explanation of that in the github docs